python/python/glide/async_commands/server_modules/json.py

```diff
@@ -320,6 +320,61 @@ async def forget(
     )
 
 
+async def resp(
+    client: TGlideClient, key: TEncodable, path: Optional[TEncodable] = None
+) -> TJsonUniversalResponse[
+    Optional[Union[bytes, int, List[Optional[Union[bytes, int]]]]]
+]:
```

Comment:
Response could be anything. It is `T`, where `T` is `Union[bytes, int, List[T], Dict[T], Optional[T]]` or so.
Can function return None? If yes, wrap `Union` above with `Optional` (and don't forget to show such example and test such case).

Reply:
Why dict ?

Reply:
The optional is within the list itself :)

Reply:
If you query `meow` element from `"meow": { "one" : 1, "two": "two", "three": null, "four": false }`, you'll get a dict. This command can return anything. I think you need to use `TResult` there as a response type.

Reply:
But this isn't what real returns 🙁 it still returns a list with the first element being {

Reply:
Oh, my mistake.
We need to drop returned `{` or `[` and convert the data

Reply:
disagree, this is a resp commands, the point is to describe the type.
if the user wants the actual data converted, use` json.get`